go.mod

```diff
@@ -62,17 +64,17 @@ replace (
 	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.7.1
 	github.com/spf13/cobra => github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag => github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify => github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify => github.com/stretchr/testify v1.6.1
 	go.uber.org/zap => go.uber.org/zap v1.14.1 // indirect
 	golang.org/x/lint => golang.org/x/lint v0.0.0-20201208152925-83fdc39ff7b5 // indirect
-	golang.org/x/oauth2 => golang.org/x/oauth2 v0.0.0-20210112200429-01de73cf58bd
+	golang.org/x/oauth2 => golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
```

Comment:
Why are we rolling to an older version of oauth2?

Reply:
Thanks for reviewing. Version changes from root go.mod have been reverted.